Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/CoronaTestProvider.kt

```diff
@@ -0,0 +1,63 @@
+package de.rki.coronawarnapp.coronatest
+
+import de.rki.coronawarnapp.coronatest.type.BaseCoronaTest
+import de.rki.coronawarnapp.coronatest.type.PersonalCoronaTest
+import de.rki.coronawarnapp.coronatest.type.TestIdentifier
+import de.rki.coronawarnapp.coronatest.type.pcr.notification.PCRTestResultAvailableNotificationService
+import de.rki.coronawarnapp.familytest.core.model.FamilyCoronaTest
+import de.rki.coronawarnapp.familytest.core.repository.FamilyTestRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.map
+import timber.log.Timber
+import javax.inject.Inject
+
+class CoronaTestProvider @Inject constructor(
+    private val coronaTestRepository: CoronaTestRepository,
+    private val familyTestRepository: FamilyTestRepository,
+    private val testResultAvailableNotificationService: PCRTestResultAvailableNotificationService,
+) {
+
+    private val allTests: Flow<Set<BaseCoronaTest>> = combine(
```

Comment:
Add shareLatest

Reply:
is is private , used internally , no need to share , I actually I added it